src/main/webapp/availabilityTable.jsp

```diff
@@ -20,23 +20,37 @@
 <table class="table table-sm text-center">
   <thead>
     <tr>
+      <th scope="col"></th>
       <c:forEach items = "${pageScope.list}" var = "day">
         <th scope="col">${day.get(0).date()}</th>
       </c:forEach>
     </tr>
   </thead>
   <tbody>
     <!-- TODO: Allow clicking and scrolling over multiple slots to select them.-->
-    <!-- TODO: Change page format so that it is vertically condensed.-->
     <c:forEach var = "i" begin = "0" end = "${pageScope.list.get(0).size() - 1}">
-      <tr>
-        <c:forEach items = "${pageScope.list}" var = "day">
-          <td onclick="toggleTile(this)" data-utc="${day.get(i).utcEncoding()}" 
-              class="${day.get(i).getClassList()}">
-            ${day.get(i).time()}
-          </td>
-        </c:forEach>
-      </tr>
+      <c:choose>
+        <c:when test="${i%4 == 0}">
+          <tr class='tall'>
+            <td>${pageScope.list.get(0).get(i).time()}</td>
+            <c:forEach items = "${pageScope.list}" var = "day">
+              <td onclick="toggleTile(this)" data-utc="${day.get(i).utcEncoding()}" 
+                  class="${day.get(i).getClassList()}">
+              </td>
```

Comment:
Code duplication but l guess ternary doesn't work?

Reply:
Yeah, this is how JSP handles conditionals.